components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationMgtDBQueries.java

```diff
@@ -95,28 +95,28 @@ public class ApplicationMgtDBQueries {
 
     public static final String LOAD_APP_COUNT_BY_TENANT_AND_FILTER = "SELECT COUNT(DISTINCT(SP_APP.APP_NAME)) FROM " +
             "SP_APP LEFT JOIN SP_INBOUND_AUTH ON SP_APP.ID = SP_INBOUND_AUTH.APP_ID WHERE SP_APP.TENANT_ID = ? " +
-            "AND SP_APP.APP_NAME != ? AND (%s)";
+            "AND SP_APP.APP_NAME != ? %s AND (%s)";
     public static final String LOAD_APP_BY_TENANT_AND_NAME = "SELECT ID, APP_NAME, VERSION, DESCRIPTION, UUID, " +
             "IMAGE_URL, ACCESS_URL, IS_DISCOVERABLE, USERNAME, USER_STORE, TENANT_ID FROM SP_APP WHERE " +
             "TENANT_ID = :TENANT_ID; AND APP_NAME = :APP_NAME;";
 
     // Load application basic information for listing with pagination
     public static final String LOAD_APP_NAMES_BY_TENANT_MYSQL = "SELECT ID, APP_NAME, VERSION, DESCRIPTION, " +
             "UUID, IMAGE_URL, ACCESS_URL, USERNAME, USER_STORE, TENANT_ID FROM SP_APP WHERE TENANT_ID = ? AND " +
-            "APP_NAME != ? ORDER BY ID DESC LIMIT ?, ?";
+            "APP_NAME != ? %s ORDER BY ID DESC LIMIT ?, ?";
```

Comment:
`SPAPP` part in `AND SP_APP.APP_NAME NOT IN (%s)` might not be the correct syntax here. Please check.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6026/commits/390d2fe79c6ef33c9fb2b82d0c545d835823a303